### PR TITLE
VLCPlaybackController: Rename to VLCPlaybackService

### PR DESCRIPTION
**Pull request by Mikanbu**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md)
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

VLCPlaybackController: Rename to VLCPlaybackService

### CODE REVIEW

File: Apple-TV/VLCCloudServicesTVViewController.m [hunk 1]
```diff
@@ -45,8 +45,8 @@ - (void)viewDidLoad {
     }
 
 //    _oneDriveController = [VLCOneDriveController sharedInstance];
-    _boxController = [VLCBoxController sharedInstance];
-    [_boxController startSession];
+//    _boxController = [VLCBoxController sharedInstance];
+//    [_boxController startSession];
```
Comment (carolanitz) [inline Apple-TV/VLCCloudServicesTVViewController.m]: why is this commented out?

File: Sources/MediaCategories/MediaCategoryViewController.swift [hunk 2]
```diff
@@ -290,7 +290,9 @@ class MediaCategoryViewController: UICollectionViewController, UICollectionViewD
         let mediaObject = isSearching ? searchDataSource.objectAtIndex(index: indexPath.row) : model.anyfiles[indexPath.row]
         if let media = mediaObject as? VLCMLMedia {
             // FIXME: This should be done in the VModel, workaround for the release.
-            services.medialibraryService.requestThumbnail(for: media)
+            if media.type() == .video {
+                services.medialibraryService.requestThumbnail(for: media)
+            }
```
Comment (carolanitz) [inline Sources/MediaCategories/MediaCategoryViewController.swift]: I believe this was merged in a separate PR ?

**Comment by Mikanbu** [comment]
Edited PR to include only the renaming.

**Comment by Mikanbu** [comment]
Rebased to master.

**Pull request merged**